specs/electra/beacon-chain.md

```diff
@@ -640,9 +652,14 @@ def queue_excess_active_balance(state: BeaconState, index: ValidatorIndex) -> No
     if balance > MIN_ACTIVATION_BALANCE:
         excess_balance = balance - MIN_ACTIVATION_BALANCE
         state.balances[index] = MIN_ACTIVATION_BALANCE
-        state.pending_balance_deposits.append(
-            PendingBalanceDeposit(index=index, amount=excess_balance)
-        )
+        validator = state.validators[index]
+        state.pending_deposits.append(PendingDeposit(
+            pubkey=validator.pubkey,
+            withdrawal_credentials=validator.withdrawal_credentials,
+            amount=excess_balance,
+            signature=bls.G2_POINT_AT_INFINITY,
+            slot=GENESIS_SLOT,
```

Comment:
I think we should define meaningful constants for both `bls.G2_POINT_AT_INFINITY` and `GENESIS_SLOT`. How about aliasing them as
- `UNSET_SIGNATURE` and
- `UNSET_DEPOSIT_SLOT`

By seeing only the point at infinity and the genesis slot, the readers wouldn't know what they mean.